source/guides/scheduler-gpu-requirements.md

```diff
@@ -0,0 +1,44 @@
+# Does the Dask scheduler need a GPU?
+
+A common question from users deploying Dask clusters is whether the scheduler has different minimum requirements to the workers. This question is compounded when using RAPIDS and GPUs.
+
+```{warning}
+This guide outlines our current advice on scheduler hardware requirements, but this may be subject to change.
+```
+
+**TLDR; It is strongly suggested that your Dask scheduler has matching hardware/software capabilities to the other components in your cluster.**
+
+Therefore, if your workers have GPUs and the RAPIDS libraries installed we recommend that your scheduler does too.
+
+## What does the scheduler use a GPU for?
+
+The Dask client generates a task graph of operations that it wants to be performed and serializes any data that needs to be sent to the workers. The scheduler handles allocating those tasks to the various Dask workers and passes serialized data back and forth. The workers deserialize the data, perform calculations, serialize the result and pass it back.
+
+This can lead users to logically ask if the scheduler needs the same capabilities as the workers/client. It doesn't handle the actual data or do any of the user calculations, it just decides where work should go.
+
+Taking this even further you could even ask "Does the Dask scheduler even need to be written in Python?". Some folks even [experimented with a Rust implementation of the scheduler](https://github.com/It4innovations/rsds) a couple of years ago.
+
+There are two primary reasons why we recommend that the scheduler has the cam capabilities:
+
+- There are edge cases where the scheduler does deserialize data.
+- Some scheduler optimizations require high-level graphs to be pickled on the client and unpickled on the scheduler.
+
+If your workload doesn't trigger any edge-cases and you're not using the high-level graph optimizations then you could likely get away with not having a GPU. But it is likely you will run into problems eventually and the failure-modes will be potentially hard to debug.
+
+### Known edge cases
+
+When calling `dask.submit` and passing data directly to a function the whole graph is serialized and sent to the scheduler. In order for the scheduler to figure out what to do with it the graph is deserialized. If the data uses GPUs this can cause the scheduler to import RAPIDS libraries, attempt to instantiate a CUDA context and populate the data into GPU memory. If those libraries are missing and/or there are no GPUs this will cause the scheduler to fail.
```

Comment:
```suggestion
When calling `client.submit` and passing data directly to a function the whole graph is serialized and sent to the scheduler. In order for the scheduler to figure out what to do with it the graph is deserialized. If the data uses GPUs this can cause the scheduler to import RAPIDS libraries, attempt to instantiate a CUDA context and populate the data into GPU memory. If those libraries are missing and/or there are no GPUs this will cause the scheduler to fail.
```